modules/logging-loki-memberlist-ip.adoc

```diff
@@ -8,7 +8,12 @@
 
 In an OpenShift cluster, the administrators generally use a non-private IP network range. As a result, the LokiStack memberlist configuration fails as it only uses private IP networks.
```

Comment:
I'd suggest to change `the LokiStack memberlist configuration fails as it only uses private IP networks.` by `the LokiStack memberlist configuration fails as, by default, it only uses private IP networks.`

Reply:
Made this change!